src/Workspaces/Core/Portable/Workspace/IWorkspaceConfigurationService.cs

```diff
@@ -43,7 +43,8 @@ internal sealed class DefaultWorkspaceConfigurationService() : IWorkspaceConfigu
 #else
         false,
 #endif
-    [property: DataMember(Order = 4)] bool RunSourceGeneratorsInSameProcessOnly = false)
+    [property: DataMember(Order = 4)] bool RunSourceGeneratorsInSameProcessOnly = false,
+    [property: DataMember(Order = 5)] bool DisableNullableAnalysisInClassification = false)
```

Comment:
Rather than this living in IWorkspaceConfigurationService, do we want to put this some classification options intead?